ipa-core/src/protocol/prss/mod.rs

```diff
@@ -106,7 +122,7 @@ impl From<u32> for PrssIndex {
 // It would be nice for this to be TryFrom, but there's a lot of places where we use u128s as PRSS indexes.
 impl From<u128> for PrssIndex {
     fn from(value: u128) -> Self {
-        Self(value.try_into().unwrap())
+        Self(value.try_into().expect("PRSS index out of range"))
```

Comment:
I know it is annoying but it would simplify the debugging if we print the actual value that is out of range

Reply:
```suggestion
        let Some(v) = u32::try_from(value) else {
            panic!("PRSS indices need to be smaller: {index} > u32::MAX");
        };
        Self(v)
```